.gitignore

```diff
@@ -8,6 +8,7 @@ pids
 *.seed
 
 .idea
+.DS_Store
```

Comment:
I recommend creating a Global `.gitignore` file for the future 😉 .
See https://help.github.com/articles/ignoring-files/#create-a-global-gitignore